docs/install_guides/installing-openvino-macos.md

```diff
@@ -1,212 +1,156 @@
 
-1. Go to the directory where you downloaded the Intel® Distribution of OpenVINO™ toolkit. This document assumes this is your `Downloads` directory. By default, the disk image file is saved as `m_openvino_toolkit_p_<version>.dmg`.
+3. Double-click the `m_openvino_toolkit_p_<version>.dmg` file to mount. The disk image is mounted to `/Volumes/m_openvino_toolkit_p_<version>` and automatically opens in a separate window.
 
-2. Double-click the `m_openvino_toolkit_p_<version>.dmg` file to mount.
-The disk image is mounted to `/Volumes/m_openvino_toolkit_p_<version>` and automatically opens in a separate window.
+4. Run the installation wizard application `bootstrapper.app`.
 
-3. Run the installation wizard application `m_openvino_toolkit_p_<version>.app`.
-
-4. On the **User Selection** screen, choose a user account for the installation:
-    - Root
-    - Administrator
-    - Current user
-
-   The default installation directory path depends on the privileges you choose for the installation.
-
-5. Follow the instructions on your screen. Watch for informational messages such as the following in case you must complete additional steps:
-   ![](../img/openvino-install-macos-02.png)
+5. Follow the instructions on your screen. During the installation you will be asked to accept the license agreement. The acceptance is required to continue.
 
 6. The **Installation summary** screen shows you the default component set to install. By default, the Intel® Distribution of OpenVINO™ is installed in the following directory, referred to as `<INSTALL_DIR>` elsewhere in the documentation:
 
    * For root or administrator: `/opt/intel/openvino_<version>/`
    * For regular users: `~/intel/openvino_<version>/`
 
-   For simplicity, a symbolic link to the latest installation is also created: `<INSTALL_DIR>/intel/openvino_2021/`.
-
-7. **Optional**: You can choose **Customize** to change the installation directory or the components you want to install.
-> **NOTE**: If there is an OpenVINO™ toolkit version previously installed on your system, the installer will use the same destination directory for next installations. If you want to install a newer version to a different directory, you need to uninstall the previously installed versions.
-
-8. The **Finish** screen indicates that the core components have been installed:
-   ![](../img/openvino-install-macos-05.png)
-
-   Once you click **Finish** to close the installation wizard, a new browser window opens with the document you’re reading now (in case you installed without it) and jumps to the section with the next installation steps.
+   For simplicity, a symbolic link to the latest installation is also created: `<INSTALL_DIR>/intel/openvino_2022/`.
 
 The core components are now installed. If you received a message that you were missing external software dependencies (list available under Software Requirements at the top of this guide), you need to install them now before continuing to the next section.
```

Comment:
This is leftover message for previous installer version. Please remove it or rework.